libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -374,6 +379,9 @@ void Aircraft::fill_fdm(struct sitl_fdm &fdm)
     fdm.scanner.points = scanner.points;
     fdm.scanner.ranges = scanner.ranges;
 
+    // copy rangefinder
+    memcpy(fdm.rangefinder_m, rangefinder_m, sizeof(rangefinder_m));
```

Comment:
Specifying the destination size is usually done as memory overwrites are usually worse that memory overreads....

```suggestion
    memcpy(fdm.rangefinder_m, rangefinder_m, sizeof(fdm.rangefinder_m));
```

Reply:
fixed